packages/react-juce/src/components/Slider.tsx

```diff
@@ -143,16 +143,16 @@ const _drawRotarySlider = (trackColor,
 }
 
 export interface SliderProps {
-  value: number
+  value?: number
   sensitivity?: number
   onChange: (value: number) => void
```

Comment:
`onChange` could be optional here too right? If you don't pass a `value` and defer the whole operation to the `Slider` itself, maybe you don't care about change callbacks?

Reply:
Yeah I wondered about that. Is it valid to have a Slider that basically does nothing?

Reply:
Totally! Maybe you're just trying to mock something up for your designer and you don't want to go the full step to wire it to any real parameters yet?

Reply:
Very good point. Coool. Optional onChange it is.